Tools/src/TrackTools.cc

```diff
@@ -25,10 +25,11 @@ namespace jmt {
     if (level >= 2) {
       jmt::TrackRescaler track_rescaler;
       if (use_rescaled) {
-        const int track_rescaler_which = jmt::TrackRescaler::w_JetHT; // JMTBAD which rescaling if ever a different one
+        const int track_rescaler_which = jmt::TrackRescaler::w_BTagDispJet; // JMTBAD which rescaling if ever a different one
```

Comment:
Let's make this a configurable option, to avoid surprises / having to fix hardcoding in the future!